[xlnt] Build error

**Host Environment**
- OS: Ubuntu 21.10
- Compiler: gcc 11.2.0

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install xlnt`

**Failure logs** 
```
/develop/vcpkg/buildtrees/xlnt/src/ea7656df91d410bfb5cd.clean/source/detail/number_format/number_formatter.cpp:1637:52: error: �~@~Xnumeric_limits�~@~Y is not a member of �~@~Xstd�~@~Y
 1637 |         result = std::fabs(fractional_part) < std::numeric_limits<double>::min()
```

**Additional context**
The file number_formatter.cpp is missing `#include <limits>` upon investigation

--- a/ports/fontconfig/no-etc-symlinks.patch
+++ b/ports/fontconfig/no-etc-symlinks.patch
@@ -1,0 +1,19 @@
+--- a/conf.d/link_confs.py	2022-03-24 04:13:59.000982000 +0900
++++ b/conf.d/link_confs.py	2022-03-24 04:14:46.271964000 +0900
+@@ -4,6 +4,7 @@
+ import sys
+ import argparse
+ import platform
++import shutil
+ 
+ if __name__=='__main__':
+     parser = argparse.ArgumentParser()
+@@ -32,7 +33,7 @@
+         except FileNotFoundError:
+             pass
+         try:
+-            os.symlink(src, dst)
++            shutil.copyfile(src, dst)
+         except NotImplementedError:
+             # Not supported on this version of Windows
+             break

--- a/ports/fontconfig/portfile.cmake
+++ b/ports/fontconfig/portfile.cmake
@@ -7,6 +7,8 @@ vcpkg_from_gitlab(
     REF ${FONTCONFIG_VERSION}
     SHA512 815f999146970c7f0e6c15a21f218d4b3f75b26d4ef14d36711bc0a1de19e59cc62d6a2c53993dd38b963ae30820c4db29f103380d5001886d55b6a7df361154
     HEAD_REF master
+    PATCHES
+        no-etc-symlinks.patch
 )
 
 vcpkg_find_acquire_program(GPERF)

--- a/ports/fontconfig/vcpkg.json
+++ b/ports/fontconfig/vcpkg.json
@@ -4,6 +4,7 @@
   "port-version": 6,
   "description": "Library for configuring and customizing font access.",
   "homepage": "https://www.freedesktop.org/wiki/Software/fontconfig",
+  "license": "MIT",
   "dependencies": [
     "dirent",
     "expat",

--- a/ports/fontconfig/vcpkg.json
+++ b/ports/fontconfig/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "fontconfig",
   "version": "2.13.94",
-  "port-version": 5,
+  "port-version": 6,
   "description": "Library for configuring and customizing font access.",
   "homepage": "https://www.freedesktop.org/wiki/Software/fontconfig",
   "dependencies": [

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2262,7 +2262,7 @@
     },
     "fontconfig": {
       "baseline": "2.13.94",
-      "port-version": 5
+      "port-version": 6
     },
     "foonathan-memory": {
       "baseline": "2019-07-21",

--- a/versions/f-/fontconfig.json
+++ b/versions/f-/fontconfig.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "d5b0b6ab59f6631a82dbd5c730501168b8e4ee25",
+      "version": "2.13.94",
+      "port-version": 6
+    },
+    {
       "git-tree": "2f32046747209c234e60dc297b48d5bdc7ee4494",
       "version": "2.13.94",
       "port-version": 5

--- a/versions/f-/fontconfig.json
+++ b/versions/f-/fontconfig.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "d5b0b6ab59f6631a82dbd5c730501168b8e4ee25",
+      "git-tree": "8f782f79dc3a2c6834a7e5de4a941b11822067dd",
       "version": "2.13.94",
       "port-version": 6
     },

